charts/prometheus-pushgateway/templates/ingress.yaml

```diff
@@ -2,7 +2,10 @@
 {{- $serviceName := include "prometheus-pushgateway.fullname" . }}
 {{- $servicePort := .Values.service.port -}}
 {{- $ingressPath := .Values.ingress.path -}}
-{{- if semverCompare ">=1.14.0-0" .Capabilities.KubeVersion.GitVersion }}
+{{- $ingressPathType := .Values.ingress.pathType -}}
```

Comment:
values should be `.Values.pushgateway.ingress` , etc..

Reply:
What makes you say that?  At least on my helm it works as I would expect as it is here:
```
% helm template --set ingress.enabled=true --set ingress.hosts='{"pushgateway.example.com"}' --set ingress.pathType=foo .  | grep pathType
            pathType: foo
% helm template --set ingress.enabled=true --set ingress.hosts='{"pushgateway.example.com"}' --set pushgateway.ingress.pathType=foo .  | grep pathType
            pathType: ImplementationSpecific
%
```